chain-test/src/test/scala/org/bitcoins/chain/blockchain/ChainHandlerTest.scala

```diff
@@ -412,6 +393,47 @@ class ChainHandlerTest extends ChainUnitTest {
       }
   }
 
+  it must "generate a range for a block filter header query" in {
+    chainHandler: ChainHandler =>
+      for {
+        bestBlock <- chainHandler.getBestBlockHeader()
+        bestBlockHashBE = bestBlock.hashBE
+        rangeOpt <- chainHandler.nextFilterHeaderBatchRange(
+          DoubleSha256DigestBE.empty,
+          1)
+      } yield {
+        assert(rangeOpt.nonEmpty)
+        assert(rangeOpt.get._1 == 0)
+        assert(rangeOpt.get._2 == bestBlockHashBE.flip)
+      }
+  }
+
+  it must "return the number of confirmations" in {
+    chainHandler: ChainHandler =>
+      for {
+        bestBlockHashBE <- chainHandler.getBestBlockHash()
+        confirmations <- chainHandler.getNumberOfConfirmations(bestBlockHashBE)
+      } yield {
+        assert(confirmations == Some(1))
```

Comment:
this can be `confirmations.contains(1)`

Reply:
I don't use `contains[A1 >: A](elem: A1)` because  the compiler can infer `A1` as `AnyVal` without warning. So `confirmations.contains("abc")` will compile, but it will always return `false` because `1 != "acb"`